eosclubhouse/system.py

```diff
@@ -679,16 +679,42 @@ def is_running(self):
         '''
         return self.get_gtk_app_proxy().props.g_name_owner is not None
 
-    def is_installed(self):
-        '''Check if the app is installed.
+    def _is_flatpak_installed(self):
+        '''Check if the app is installed as flatpak.
 
         Note: This only works for apps distributed as flatpaks.
         '''
+
         result = subprocess.run(['/usr/bin/flatpak-spawn', '--host',
                                  'flatpak', 'info', '--show-ref', self.dbus_name],
                                 capture_output=True)
         return result.returncode == 0
 
+    def _is_app_installed(self):
+        app_name = f'{self.dbus_name}.desktop'
+        sandbox = get_flatpak_sandbox()
+        # Remove /app
+        launch_script = '/'.join(IS_INSTALLED_SCRIPT_PATH.split('/')[2:])
+        script_path = f'{sandbox}/{launch_script}'
+        try:
+            subprocess.run(['/usr/bin/flatpak-spawn', '--host',
+                            script_path, app_name], check=True)
+        except Exception as e:
+            return False
+        return True
+
+    def is_installed(self):
+        '''Check if the app is installed.
+
+        Looks for the .desktop file on the system and if that file is not found
+        it check if the flatpak is installed using flatpak-spawn
+        '''
+
+        if not self._is_app_installed():
+            return self._is_flatpak_installed()
```

Comment:
The new method superceeds the previous `_is_flatpak_installed`. I. e. It will work also flatpak apps. Thus may be better to remove the old one.

Reply:
yes, you're right, we can remove that one.